src/routes/(app)/import/+page.svelte

```diff
@@ -406,7 +407,7 @@
       // Build toImport array
       const toImport: ImportedList[] = [];
       const fileText = await readFile(new FileReader(), file);
-      const jsonData = JSON.parse(fileText)["media"] as Watched[];
+      const jsonData = JSON.parse(fileText)["media"] as any[];
```

Comment:
Changed to `any` array, the data in the ryot export didnt conform to the `Watched[]` type